lib/kafo/configuration.rb

```diff
@@ -60,7 +60,9 @@ def save_configuration(configuration)
       begin
         FileUtils.touch @config_file
         File.chmod 0600, @config_file
-        File.open(@config_file, 'w') { |file| file.write(format(YAML.dump(configuration))) }
+        File.open(@config_file, 'w') do |file|
+          file.write(format(YAML.dump(configuration.sort.to_h)))
```

Comment:
What is the benefit of calling `to_h` here when it wasn't used before?

Reply:
Without the `to_h` this will write itself as an array. The sort turns the hash into a sorted array, the to_h turns it back into a hash.

Reply:
OK, I tested that and I see it's indeed the case. I read in some older tutorial that `sort` doesn't handle hashes (hence my confusion about the type of `configuration` to begin with) with symbol keys well but it's working just fine for me at the ruby console.

Any reason that you left this as a draft?

Reply:
Originally, I left it as a draft just do the amount of other PRs touching this area and wishing to get them in first. But I will switch out and deal with rebases.